src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -872,5 +872,223 @@
       "path": "type",
       "notEquals": "Microsoft.ClassicStorage/storageAccounts"
     }
+  },
+  {
+    "id": "TA-000032",
+    "name": "APIM.BypassCertificateThumbprintOrNameValidation",
+    "shortDescription": "API Management calls to API backends should not bypass certificate thumbprint or name validation",
+    "fullDescription": "API Management calls to API backends should not bypass certificate thumbprint or name validation",
+    "recommendation": "To improve security, API Management should validate the backend server certificate for all API calls. Enable SSL certificate thumbprint and name validation by setting the tls.validateCertificateName and tls.validateCertificateChain properties to `true`.",
```

Comment:
```suggestion
    "recommendation": "To improve security, API Management should validate the backend server certificate for all API calls. Enable SSL certificate thumbprint and name validation by setting the tls.validateCertificateName and tls.validateCertificateChain properties to true.",
```

Reply:
Hey Vera, same here. Should I use `` when referencing the properties?: https://github.com/Azure/template-analyzer/pull/316#discussion_r1093823805